INSTALL.md

```diff
@@ -0,0 +1,268 @@
+# Installing google-cloud-cpp
+
+By default google-cloud-cpp libraries download and compile all their
+dependencies. This makes it easier for users to "take the library for a spin",
+and works well for users that "Live at Head", but does not work for package
+maintainers or users that prefer to compile their dependencies once and install
+them in `/usr/local/` or a similar directory.
+
+This document provides instructions to install the dependencies of
+google-cloud-cpp.
+
+## Installing google-cloud-cpp
+
+*If*<br>
+-- The Spartan ephors to Philip of Macedon.<br><br>
+
+**If** all the dependencies of `google-cloud-cpp` are installed and provide
+CMake support files, then compiling and installing the libraries
+requires two commands:
+
+```bash
+cmake -H. -Bbuild-output-for-install \
+    -DGOOGLE_CLOUD_CPP_DEPENDENCY_PROVIDER=package
+cmake --build build-output-for-install --target install
+```
+
+Unfortunately getting your system to this state may require multiple steps,
+the following sections describe how to install `google-cloud-cpp` on several
+platforms.
+
+## Table of Contents
+
+- [Installing google-cloud-cpp](#installing-google-cloud-cpp)
+  - [Fedora 29](#fedora-29)
+  - [Ubuntu (Bionic Beaver)](#ubuntu-bionic-beaver)
+  - [CentOS 7](#ubuntu-bionic-beaver)
+
+## Installing google-cloud-cpp
+
+#### Required Libraries
+
+`google-cloud-cpp` directly depends on the following libraries:
+
+| Library | Minimum version | Description |
+| ------- | --------------: | ----------- |
+| gRPC    | 1.17.x | gRPC++ for Cloud Bigtable |
+| libcurl | 7.47.0  | HTTP client library for the Google Cloud Storage client |
+| crc32c  | 1.06 | Hardware-accelerated CRC32C implementation |
+| OpenSSL | 1.0.2 | Crypto functions for Google Cloud Storage authentication |
+
+Note that these libraries may have complex dependencies themselves, the
+following instructions include steps to install these dependencies too.
+
+#### Fedora (29)
+
+Fedora includes packages for gRPC (1.17), libcurl (7.61), and OpenSSL (1.1).
```

Comment:
A feature request for maintainers would be having a way to generate these README's with a single source for dependency versions. I suspect typo's will occur.  

Otherwise keep version numbers in a single place at the top in the minimum version table.

Reply:
I removed the version numbers.